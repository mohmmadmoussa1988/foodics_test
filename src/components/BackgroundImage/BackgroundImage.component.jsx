import React from 'react';
import { BackgroundImageWrapper, OverlayWrapper } from './BackgroundImage.styles';
const BackgroundImage = () => {
    return (
        <BackgroundImageWrapper>
            <OverlayWrapper></OverlayWrapper>
        </BackgroundImageWrapper>
    )
}

export default BackgroundImage;