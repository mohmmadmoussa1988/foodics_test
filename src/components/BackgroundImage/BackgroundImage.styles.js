import styled from "styled-components";
import BackgroundImage from "../../assets/background.jpg";
export const BackgroundImageWrapper = styled.div`
  background: url(${BackgroundImage}) no-repeat;
  background-size: cover;
  background-position: center -25em;
  height: 87%;
  position: absolute;
  z-index: -1;
  top: 0;
  left: 0;
  width: 100%;
  -webkit-clip-path: polygon(
    50% 0%,
    100% 0,
    100% 35%,
    100% 49%,
    100% 75%,
    49% 85%,
    0 75%,
    0 46%,
    0 20%,
    0 0
  );
  clip-path: polygon(
    50% 0%,
    100% 0,
    100% 35%,
    100% 49%,
    100% 75%,
    49% 85%,
    0 75%,
    0 46%,
    0 20%,
    0 0
  );
`;
export const OverlayWrapper = styled.div`
  background-image: linear-gradient(
    to bottom,
    rgba(23, 55, 83, 0.1),
    rgba(4, 21, 35, 0.87)
  );
  width: 100%;
  height: 100%;
  position: absolute;
  top: 0px;
  left: 0px;
`;
