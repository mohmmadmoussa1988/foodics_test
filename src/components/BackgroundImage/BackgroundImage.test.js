import React from "react";
import { configure, shallow } from "enzyme";
import BackgroundImage from "./BackgroundImage.component";
import toJson from "enzyme-to-json";

describe(`<BackgroundImage />`, () => {
  it("should render Background image", () => {
    const wrapper = shallow(<BackgroundImage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
