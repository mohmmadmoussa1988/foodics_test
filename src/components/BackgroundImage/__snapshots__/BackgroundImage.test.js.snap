// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BackgroundImage /> should render Background image 1`] = `
<styled.div>
  <styled.div />
</styled.div>
`;
